features/api-resource-mgt/org.wso2.carbon.identity.api.resource.mgt.server.feature/resources/system-api-resource.xml.j2

```diff
@@ -27,6 +27,16 @@
         </Scopes>
     </APIResource>
     {% endfor %}
+    <APIResource name="SCIM2 Roles API"
```

Comment:
This will be changed to proper place when the issue [1] is fixed.

[1]https://github.com/wso2/product-is/issues/21306